gpioioctl/basic_test.go

```diff
@@ -0,0 +1,120 @@
+package gpioioctl
+
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
+// Basic tests. More complete test is contained in the
+// periph.io/x/cmd/v3/periph-smoketest/gpiosmoketest
+// folder.
+import (
+	"testing"
+
+	"periph.io/x/conn/v3/driver/driverreg"
+	"periph.io/x/conn/v3/gpio/gpioreg"
+)
+
+var _test_line *GPIOLine
+
+func init() {
+	_, _ = driverreg.Init()
+}
+
+func TestChips(t *testing.T) {
+	if len(Chips) <= 0 {
+		t.Fatal("Chips contains no entries.")
+	}
+	chip := Chips[0]
+	if len(chip.Name()) == 0 {
+		t.Error("chip.Name() is 0 length")
+	}
+	if len(chip.Path())==0 {
```

Comment:
You may want to configure gofmt -w -s and/or goimports -w automatically in your IDE. The github checks won't pass otherwise.